addons/core/translations/resources/en-us.yaml

```diff
@@ -571,6 +571,8 @@ user:
   actions:
     add-accounts: Add Accounts
     delete: Delete User
+    edit-details:
+      aria-description: 'Edit Form: User'
```

Comment:
I think I might have been mistaken and you might not even need this line and can just use `user.title`. The `Edit Form: User` was for an `aria-label` to make sure the `Edit Form` part matches but I think `aria-description` is similar to `aria-describedby` and we only just need to describe the title after edit form, otherwise we get this:

<img width="646" height="156" alt="Image" src="https://github.com/user-attachments/assets/664b68a0-c32d-4dbc-8b8c-5e1fd33213c1" />

where it reads `Edit form` twice